src/SDK/Client/Platform/methods/documents/get.ts

```diff
@@ -23,6 +36,8 @@ declare interface fetchOpts {
  * @returns documents
  */
 export async function get(this: Platform, typeLocator: string, opts: fetchOpts): Promise<any> {
+    await ensureAppsContracts.call(this);
```

Comment:
It's good for the first iteration, but in the future we could load a contract when it present in the locator.

Reply:
You won't always know it. 

There is two things : 

get('dpns.domain') which would work how you envision it (if dpns, then prefetch it)
get('domain') which require to know domain exist in 'dpns' prior, therefore why we prefetch it. 

It also need to deal when multiple contracts have "domain", such in a way if you have dpns and dpns2, you should know that you do not know which one to use. 

It's mainly for reason two that we do that.

Reply:
I see. `get('domain')` this works just in case when only one contract is defined?

Reply:
I'm not worries about confusion or bugs: 
1) For bugs, that's why we have tests.
2) For confusion, that's why we rejects if there is two similar fieldName.
 
However, removing that feature would indeed remove the requirement.
I do not know if community needs this feature in the first place, nor who (except us) makes that feature call. 
As far as I know, me as user, I prefer to use "dpns.domain" such you can see in "identities" implementation. 
And I did not stumble upon a community code that uses that. 

@thephez @antouhou @Cofresi : Except if you guys find we should keep the feature describe just above, we gonna removes it. Are you good with that ?

Reply:
I mean bugs on the user code. For example, initially, it was just one contract and everything fine but then we add more. After that, we could update the contract and add one document type with conflicting name. I feel it could be a source of bugs or confusion.

Yeah, I personally prefer to specify a contract too. It’s more explanatory.

Reply:
This is done ! :)